versions.env

```diff
@@ -0,0 +1,5 @@
+DB2_VERSION=11.5.9.0
+MARIADB_VERSION=10.6.22-jammy
+MYSQL_VERSION=5.7
```

Comment:
@kyle-miho this MySQL version is a downgrade from the current default. Was this intentional?